src/buildApplication/nextConfigs/index.ts

```diff
@@ -0,0 +1,18 @@
+import { getBasePath } from './getBasePath';
+
+export type NextConfigs = {
+	basePath?: string;
+};
+
+/**
+ * generates an object containing the next.config.js options that next-on-pages
+ * supports
+ *
+ * For the list of supported options see: __TODO: add link to support.md section after readme PR is merged__
+ */
+export async function getNextConfigs(): Promise<NextConfigs> {
```

Comment:
Yeah you're right, sorry about it, I'll check the code and try to use `NextJs` instead of `Next` 🙇 

PS: I am not sure if I prefer `Nextjs` (as a single term) 🤔, I guess you prefer `NextJs`?